### ISSUE TITLE
Smudge vs serf/memberlist

### ISSUE THREAD

**Issue by max19931** [ClockworkSoul/smudge#38]
This project looks similar to serf and memberlist from hashicorp, yours and serf are based on swim.

**Comment by eelcocramer** [comment]
You are right there are similarities. There are also some differences however. One noticeable difference is that Smudge uses only `udp` messages where serf may use `tcp` as well.
